framework_tvb/tvb/adapters/exporters/tvb_export.py

```diff
@@ -38,6 +38,7 @@
 from tvb.adapters.exporters.exceptions import ExportException
 from tvb.core.entities.model.model_datatype import DataType
 from tvb.core.neocom import h5
+from tvb.core.neotraits._h5core import H5File
```

Comment:
Importing modules with underscore is not advices outside their own package. Can't you find h5file in "api"?